Cargo.toml

```diff
@@ -1,23 +1,19 @@
 [package]
 name = "indexer-explorer"
-version = "0.10.14"
+version = "0.10.15"
 authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2021"
 
 [dependencies]
-actix = "=0.11.0-beta.2"
-actix-rt = "=2.2.0"  # remove it once actix is upgraded to 0.11+
-actix-web = "=4.0.0-beta.6"
-actix-http = "=3.0.0-beta.6"
-actix-tls = "=3.0.0-beta.5"
-actix_derive = "=0.6.0-beta.1"
+actix = "=0.13.0"
+actix-rt = "=2.2.0"
```

Comment:
Well, you removed the comment, but do we actually still need to pin actix-rt version? I would imagine we can drop this line completely

Reply:
I wanted to remove it, but we're using it

https://github.com/near/near-indexer-for-explorer/blob/3f1584b5a74236f89d89989f85db11c7d691a119/src/db_adapters/genesis.rs#L22-L25

I have unpinned the version